compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1777,7 +1778,11 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
     }
 
     fn lower_lifetime(&mut self, l: &Lifetime) -> &'hir hir::Lifetime {
-        self.new_named_lifetime(l.id, l.id, l.ident)
+        self.new_named_lifetime(l.id, l.id, l.ident, IsPathAnon::No)
+    }
+
+    fn lower_lifetime_is_path_anon(&mut self, id: NodeId, span: Span) -> &'hir hir::Lifetime {
```

Comment:
```suggestion
    fn lower_lifetime_path_anon(&mut self, id: NodeId, span: Span) -> &'hir hir::Lifetime {
```
or `lower_lifetime_in_path_anon` or sth 😅 i'd expect a function with "_is_X" to have an argument to set X

Reply:
I'll do `lower_lifetime_anon_in_path` and `IsAnonInPath` for the typename.